src/components/useraccount/orderdetail/ShipmentInfo.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div v-if="me">
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('shippingAddress') }}</div>
+          <BaseAddress v-if="me.order.shippingAddress"
+                       :address="me.order.shippingAddress"/>
+        </div>
+      </div>
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('billingAddress') }}</div>
+          <BaseAddress v-if="me.order.billingAddress"
+                       :address="me.order.billingAddress"/>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('shippingMethod') }}</div>
+          <div>
+            {{ me.order.shippingInfo.shippingMethod.name }}
+            <span v-if=" me.order.shippingInfo.shippingMethod.description">
+              - {{ me.order.shippingInfo.shippingMethod.description }}
```

Comment:
I think I already mentioned that in another comment: use computed property to avoid repeating `me.order.shippingInfo.shippingMethod`.

Reply:
And also repeated: shippingInfo might not exist. Please check always optional fields.